src/utils/util/resolver-pipeline/testing/3_merged/merge.xspec

```diff
@@ -36,95 +76,657 @@
+                    </modify>
+                </catalog>
+            </x:expect>
+        </x:scenario>
+        <x:scenario label="Catalog with back matter inside and outside selection">
+            <x:context>
+                <catalog uuid="zzz">
+                    <selection uuid="xyz-tiny_catalog">
+                        <control id="x1"><title>Control X1</title></control>
+                        <back-matter>
+                            <resource uuid="0050231f-4fd0-43d6-8fa0-431367cd83e2">
+                                <title>Ref BBB</title>
+                                <citation>
+                                    <text>Bbb bbbbb Bbb b.</text>
+                                </citation>
+                                <rlink href="bbb.html"/>
+                            </resource>
+                        </back-matter>
+                    </selection>
+                    <!-- no merge -->
+                    <back-matter>
+                        <resource uuid="0050231f-4fd0-43d6-8fa0-431367cd83e1">
+                            <title>Ref AAA</title>
+                            <citation>
+                                <text>Aaaa aaaa aaaa</text>
+                            </citation>
+                            <rlink href="aaa.html"/>
+                        </resource>
+                    </back-matter>
+                </catalog>
+            </x:context>
+            <x:expect label="Output includes consolidated back matter">
+                <catalog uuid="zzz">
+                    <control id="x1"><title>Control X1</title></control>
+                    <back-matter>
+                        <resource uuid="0050231f-4fd0-43d6-8fa0-431367cd83e2">
+                            <title>Ref BBB</title>
+                            <citation>
+                                <text>Bbb bbbbb Bbb b.</text>
+                            </citation>
+                            <rlink href="bbb.html"/>
+                        </resource>
+                        <resource uuid="0050231f-4fd0-43d6-8fa0-431367cd83e1">
+                            <title>Ref AAA</title>
+                            <citation>
+                                <text>Aaaa aaaa aaaa</text>
+                            </citation>
+                            <rlink href="aaa.html"/>
+                        </resource>
+                    </back-matter>
+                </catalog>
+            </x:expect>
+        </x:scenario>
+        <x:scenario label="as-is element appears but content is not true">
+            <x:variable name="ov:as-is-content">0</x:variable>
+            <x:like label="SHARED: as-is element with placeholder for content"/>
+        </x:scenario>
+        <x:scenario label="as-is element appears but has no content">
+            <x:variable name="ov:as-is-content"/>
+            <x:like label="SHARED: as-is element with placeholder for content"/>
+        </x:scenario>
```

Comment:
FYI: x:variable and x:like make a good pair if you find yourself doing a lot of copy-paste-modify operations in an XSpec file. See also x:expand-text on line 292, which helps with parameterizing the shared content.